packages/Chem/src/chem-searches.ts

```diff
@@ -230,30 +224,101 @@ export async function chemFindSimilar(molStringsColumn: DG.Column, queryMolStrin
 }
 
 export async function chemSubstructureSearchLibrary(
-  molStringsColumn: DG.Column, molString: string, molBlockFailover: string, usePatternFingerprints = false)
-  : Promise<DG.BitSet> {
+  molStringsColumn: DG.Column, molString: string, molBlockFailover: string, columnIsCanonicalSmiles = false, awaitAll = true
```

Comment:
This function signature should be documented.

Reply:
Done